Assets/MixedRealityToolkit/Utilities/MixedRealityPlayspace.cs

```diff
@@ -14,21 +15,32 @@
 namespace Microsoft.MixedReality.Toolkit
 {
     /// <summary>
-    /// A static class encapsulating the Mixed Reality playspace.
+    /// A class encapsulating the Mixed Reality playspace.
     /// </summary>
-    public static class MixedRealityPlayspace
+    [ExecuteAlways]
+    public partial class MixedRealityPlayspace : MonoBehaviour
```

Comment:
What's the benefit of turning this into a `MonoBehaviour` with a scene requirement? It's been working well so far without that requirement.

Reply:
It works fine when you're only dealing with one scene and one playspace transform. But once you've got multiple scenes with multiple playspace transforms, a MonoBehaviour makes searching for / switching between active playspace transforms a lot simpler. Plus you can display in-editor documentation with a custom inspector so people understand what's going on.